_includes/content-toggle.html

```diff
@@ -32,47 +32,3 @@
         {% endfor %}
     </div>
 </div>
-
-<script>
-    jqueryUiDefer(initContentToggle{{contentToggleId}});
-    function initContentToggle{{contentToggleId}}() {
-        $(function(){
-
-            window.addEventListener('popstate', onPopState{{contentToggleId}});
-
-            onPopState{{contentToggleId}}();
-
-            $('.tb-content-toggle#{{contentToggleId}} a.content-toggle-button').click(function(event) {
-                event.preventDefault();
-                var id = $(this).attr("data-target").substring(1);
-                var param = '{{contentToggleId}}';
-                var params = Qs.parse(window.location.search, { ignoreQueryPrefix: true });
-                params[param] = id;
-
-                var newurl = window.location.protocol + "//" + window.location.host + window.location.pathname + '?' + Qs.stringify(params);
-                if (window.location.hash) {
-                    newurl += window.location.hash;
-                }
-                window.history.pushState({ path: newurl }, '', newurl);
-                selectTarget{{contentToggleId}}(id);
-            });
-        });
-    }
-
-    function onPopState{{contentToggleId}}() {
-        var params = Qs.parse(window.location.search, { ignoreQueryPrefix: true });
-        var targetId = params['{{contentToggleId}}'];
-        if (!targetId) {
-            targetId = '{{firstTargetId}}';
```

Comment:
Where is variable "firstTargetId" initialized? We may need to delete unnecessary variables.